lighthouse/blueprints/plates.py

```diff
@@ -22,109 +29,94 @@
 
 
 @bp.route("/plates/new", methods=["POST"])
-def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
+def create_plate_from_barcode() -> FlaskResponse:
+    """This endpoint attempts to create a plate in Sequencescape.
+
+    Returns:
+        FlaskResponse: the endpoints acts as proxy and returns the response and status code received from Sequencescape.
+    """
+    logger.info("Attempting to create a plate in Sequencescape")
     try:
         barcode = request.get_json()["barcode"]
-        logger.info(f"Attempting to create a plate in SS from barcode: {barcode}")
     except (KeyError, TypeError) as e:
         logger.exception(e)
-        return {"errors": ["POST request needs 'barcode' in body"]}, HTTPStatus.BAD_REQUEST
+
+        return bad_request("POST request needs 'barcode' in body")
 
     try:
         # get samples for barcode
-        samples = get_positive_samples(barcode)
+        samples = get_fit_to_pick_samples(barcode)
 
         if not samples:
-            return {"errors": ["No samples for this barcode: " + barcode]}, HTTPStatus.BAD_REQUEST
+            return bad_request(f"No samples for this barcode: {barcode}")
 
         # add COG barcodes to samples
         try:
             centre_prefix = add_cog_barcodes(samples)
-        except (Exception) as e:
+        except Exception as e:
+            msg = f"{ERROR_PLATES_CREATE} {ERROR_ADD_COG_BARCODES} {barcode}"
+            logger.error(msg)
             logger.exception(e)
-            return (
-                {"errors": ["Failed to add COG barcodes to plate: " + barcode]},
-                HTTPStatus.BAD_REQUEST,
-            )
+
+            return bad_request(msg)
 
         body = create_post_body(barcode, samples)
 
-        response = send_to_ss(body)
+        response = send_to_ss_heron_plates(body)
 
-        if response.ok:
+        if response.status_code == HTTPStatus.OK:
             response_json = {
                 "data": {
                     "plate_barcode": samples[0][FIELD_PLATE_BARCODE],
                     "centre": centre_prefix,
-                    "number_of_positives": len(samples),
+                    "number_of_fit_to_pick": len(samples),
                 }
```

Comment:
This requires changes in lighthouse-ui too

Reply:
Changed that too.

Reply:
Where is the Lighthouse UI change, in GPL-843 ?

Reply:
Yeah I think I pushed that up to the same PR. Will double check.